### PR TITLE
issue 1249: added a mark that the filter has been applied

### PR DESCRIPTION
**Pull request by Iogsotot**
added a mark that the filter has been applied
https://github.com/jdi-testing/jdn-ai/issues/1249

https://github.com/jdi-testing/jdn-ai/assets/50149163/7fafd98e-078d-4d01-9d77-d4f770491a5b

### CODE REVIEW

**Review by kostrubin — APPROVED** [review]

File: src/features/filter/Filter.tsx [hunk 1]
```diff
@@ -21,6 +22,8 @@ export const Filter = () => {
   const areSelectedAll = useSelector(selectIfSelectedAll);
   const classFilterArr = useMemo(() => convertFilterToArr(classFilter, searchTerm), [classFilter, searchTerm]);
 
+  const isFiltered = checkFilterArrayForFalse(classFilterArr);
```
Comment (MariiaNebesnova) [inline src/features/filter/Filter.tsx:25]: common approach on the project - is to use selectors for such calculations

File: src/features/filter/Filter.tsx [hunk 2]
```diff
@@ -65,6 +68,24 @@ export const Filter = () => {
     setOpen(flag);
   };
 
+  const filterButton = useMemo(
```
Comment (MariiaNebesnova) [inline src/features/filter/Filter.tsx:71]: for functions, that returns jsx, please, start name with "render"

File: src/features/filter/Filter.tsx [hunk 3]
```diff
@@ -65,6 +68,24 @@ export const Filter = () => {
...
+            <Badge dot={true} color="blue" offset={[1, 4]}>
+              <Funnel size={14} color="#8C8C8C" />{" "}
+            </Badge>
+          }
+        />
+      ) : (
+        <Button className="jdn__filter_filter-button" type="link" icon={<Funnel size={14} color="#8C8C8C" />} />
+      ),
+    [isFiltered]
+  );
```
Comment (MariiaNebesnova) [inline src/features/filter/Filter.tsx:87]: I believe, that if ternary will be moved inside `icon={...}`, there will be less code duplication. Also `<Funnel size={14} color="#8C8C8C" />` can be added to a constant and reused

**Review by MariiaNebesnova — APPROVED** [review]
Thank for code!
Version up, please, and merge

**Pull request merged**